### PR TITLE
Zenden/5 homepage partners

### PR DESCRIPTION
**Pull request by zendenleong**
Added responsiveness, background image, bg colour and opacity to homepage partners header

### CODE REVIEW

File: components/home/Partners.tsx [hunk 1]
```diff
@@ -10,6 +10,7 @@ import { Box, Text } from '@chakra-ui/react'
 const Partners = ({ partnersImages }: PartnersProps) => {
   return (
     <div className="section-my">
+      <h3 className='absolute m-12 mt-[80px] hidden md:block md:ml-[475px] lg:ml-[710px] xl:ml-[950px]  text-xl text-white max-w-lg '>Past and Present</h3>
```
Comment (JustinPeng13) [inline components/home/Partners.tsx:13]: Could we use `ml-auto` instead to keep the text aligned to the right?

File: components/home/Partners.tsx [hunk 2]
```diff
@@ -4,13 +4,22 @@ import { PartnersProps } from '@interfaces/home/Partners';
...
     <div className="section-my">
-      <h2 className="mb-16 text-center text-h2 font-bold">
-        Past and Present Partners
-      </h2>
+      <h3 className='absolute m-12 mt-[80px] hidden md:block md:ml-[475px] lg:ml-[710px] xl:ml-[950px]  text-xl text-white max-w-lg '>Past and Present</h3>
+      <div className='w-full flex flex-wrap rounded-[20px] p-8 mb-8 '>
+        <div className='w-full h-[122px] flex flex-wrap bg-[#004F5080] rounded-[20px]'>
+          <img src='/images/home/ourpartners.png' className= 'w-full h-full rounded-[20px]' />
+        </div>
+        <div className='absolute flex m-[70px] mt-[40px] text-4xl text-center sm:text-5xl sm:mt-8 font-bold text-white'>
```
Comment (JustinPeng13) [inline components/home/Partners.tsx:18]: Similarly, try to use `mt-10` instead of `mt-[40px]`

**Review by JustinPeng13 — CHANGES_REQUESTED** [review]

File: components/home/Partners.tsx (hunk 2 above)
Comment (zendenleong) [reply components/home/Partners.tsx]: I have referenced your code for the header instead!

File: components/home/Partners.tsx (hunk 1 above)
Comment (zendenleong) [reply components/home/Partners.tsx]: done!

**Review by JustinPeng13 — APPROVED** [review]

**Pull request merged**